config/kdump.py

```diff
@@ -1,44 +1,69 @@
-import os
+import sys
+
 import click
 import utilities_common.cli as clicommon
-from swsscommon.swsscommon import ConfigDBConnector
+from swsssdk import ConfigDBConnector
+
 
 @click.group(cls=clicommon.AbbreviationGroup, name="kdump")
 def kdump():
-    """ Configure kdump """
-    if os.geteuid() != 0:
-        exit("Root privileges are required for this operation")
+    """Modify configuration of kdump"""
+    pass
+
 
 @kdump.command()
 def disable():
-    """Disable kdump operation"""
+    """Disable kdump feature"""
     config_db = ConfigDBConnector()
-    if config_db is not None:
+    if config_db:
```

Comment:
>if [](http://example.com/codeflow?start=4&length=2)

If the condition is false, you silently exit. Is it an error?

Reply:
Good catch! If this condition is false, we need show an error message to user instead of exiting silently.